app/celery/provider_tasks.py

```diff
@@ -189,3 +189,22 @@ def send_email_to_provider(self, service_id, notification_id):
         statsd_client.timing("notifications.tasks.send-email-to-provider.task-time", monotonic() - task_start)
         delta_milliseconds = (datetime.utcnow() - notification.created_at).total_seconds() * 1000
         statsd_client.timing("notifications.email.total-time", delta_milliseconds)
+
+
+def get_html_email_renderer(service):
+    govuk_banner = service.branding != BRANDING_ORG
+    if service.organisation:
+        logo = '{}{}{}'.format(
+            current_app.config['ADMIN_BASE_URL'],
```

Comment:
It would be nice if localhost worked the same as other environments, but this is a case where it probably does not matter.